SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc

```diff
@@ -0,0 +1,326 @@
+
+|PID-8/IS-1
+|Administrative Sex
+|pm:PatientContextState
+/pm:CoreData
+/pm:Sex
+|Note that the HL7 Administrative Sex value set (HL7 table 0001) differs from the SDC pm:Sex value set and requires a mapping accordingly (see also <<ref_tbl_sex_mapping>>).
+
+|===
+
+[#ref_tbl_sex_mapping]
+.Patient's Sex Value Set Mapping
+|===
+|SDC Value |SDC Description |HL7 Value |HL7 Description
+
+|Unspec
+|Unspecified. Sex is not designated.
+|A
+|Ambiguous
+
+|M
+|Male. Indicates a male patient.
+|M
+|Male
+
+|F
+|Female. Indicates a female patient.
+|F
+|Female
+
+|Unkn
+|Unknown. Indicates that the sex is unknown for different reasons.
+|U
+|Unknown
+
+|===
+
+// https://build.fhir.org/ig/HL7/uv-pocd/StructureDefinition-Patient.html
+
+====== PID-10 Race
+The PID-10 field shall contain the patient's race. The table <<ref_tbl_pid10_mapping>> defines the mapping of the SDC patient's race information to the data fields of the HL7 data type *CWE* used in the PID-10 field.
+
+[#ref_tbl_pid10_mapping]
+.PID-10 Race Mapping
+|===
+|HL7 Field |HL7 Component Name |SDC Attribute/Element |Comments
+
+|PID-10/CWE-1
+|Identifier
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/@Code
+|
+
+|PID-10/CWE-2
+|Text
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/@SymbolicCodeName
+|
+
+|PID-10/CWE-3
+|Name of Coding System
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/@CodingSystem
+|
+
+|PID-10/CWE-4
+|Alternate Identifier
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/pm:Translation
+/@Code
+|Note that only the first entry of the *pm:Translation* element list shall be mapped.
+
+|PID-10/CWE-6
+|Name of Alternate Coding System
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/pm:Translation
+/@CodingSystem
+|Note that only the first entry of the *pm:Translation* element list shall be mapped.
+
+|PID-10/CWE-7
+|Coding System Version ID
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/@CodingSystemVersion
+|
+
+|PID-10/CWE-8
+|Alternate Coding System Version ID
+|pm:PatientContextState
+/pm:CoreData
+/pm:Race
+/pm:Translation
+/@CodingSystemVersion
+|Note that only the first entry of the *pm:Translation* element list shall be mapped.
+
+|===
+
+====== PID-31 Identity Unknown Indicator
+The PID-31 field shall indicate whether or not the patient's identity is known.
+
+The field value shall be determined from the present of a patient-inferred ensemble context (*pm:EnsembleContextDescriptor/pm:Type/@Code = MDC_IDT_ENSEMBLE_PATIENT_INFERRED*) with an associated context association state (*pm:EnsembleContextState/@ContextAssociation = Assoc*). In this case, the value shall be set to "Y".
+
+In all other cases, the value shall be set to "N".
+
+====== PID-33 Last Update Date/Time
+The PID-33 field shall contain the date & time of last patient context update. The table <<ref_tbl_pid33_mapping>> defines the mapping of the SDC patient context update information to the data fields of the HL7 data type *DTM* used in the PID-33 field.

```

Comment:
For the sake of not tainting the review with "please make this a requirement box" any further:

- Each _shall_ should be transformed to a requirement box, esp. to be able to link it to other specs in the long run (RI).
- Alternatively, I think some of these requirement texts could also be phrased as definitions, which are mandatory to be implemented in the way they are defined. For example here: "The PID-33 field contains the date & time of last patient context update."

Reply:
Done.